src/main/java/com/synopsys/integration/alert/AboutReader.java

```diff
@@ -42,19 +46,23 @@
     private final Logger logger = LoggerFactory.getLogger(AboutReader.class);
     private final Gson gson;
     private final SystemStatusUtility systemStatusUtility;
+    private final DescriptorMetadataActions descriptorActions;
 
     @Autowired
-    public AboutReader(Gson gson, SystemStatusUtility systemStatusUtility) {
+    public AboutReader(Gson gson, SystemStatusUtility systemStatusUtility, DescriptorMetadataActions descriptorActions) {
         this.gson = gson;
         this.systemStatusUtility = systemStatusUtility;
+        this.descriptorActions = descriptorActions;
     }
 
     public AboutModel getAboutModel() {
         try {
             String aboutJson = ResourceUtil.getResourceAsString(getClass(), "/about.txt", StandardCharsets.UTF_8.toString());
             AboutModel aboutModel = gson.fromJson(aboutJson, AboutModel.class);
             String startupDate = systemStatusUtility.getStartupTime() != null ? DateUtils.formatDate(systemStatusUtility.getStartupTime(), RestConstants.JSON_DATE_FORMAT) : "";
-            return new AboutModel(aboutModel.getVersion(), aboutModel.getCreated(), aboutModel.getDescription(), aboutModel.getProjectUrl(), systemStatusUtility.isSystemInitialized(), startupDate);
+            Set<DescriptorMetadata> providers = descriptorActions.getDescriptors(null, DescriptorType.PROVIDER.name(), null);
```

Comment:
Add the provider and channel descriptor information back into the about model like we used to in older versions of Alert.

Reply:
Note this DescriptorMetadata has no operations data contained in it.  So the UI will not display anything based on permissions if these DescriptorMetadata objects are used.

Reply:
Possibly a future improvement but should we create a `getDescriptors()` method in `DescriptorMetadataActions` that only takes in a DescriptorType name?

Reply:
Implemented your suggestion because the getDescriptorsByType is what we currently do for the about page.